src/test/kotlin/no/nav/familie/dokument/pdf/PdfGenerartorTest.kt

```diff
@@ -0,0 +1,128 @@
+package no.nav.familie.dokument.pdf
+
+import com.openhtmltopdf.pdfboxout.visualtester.PdfVisualTester
+import org.apache.commons.io.IOUtils
+import org.junit.Assert
+import org.junit.Test
+import java.io.File
+import java.io.IOException
+import java.net.URISyntaxException
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.util.stream.Collectors
+import javax.imageio.ImageIO
+
+class PdfGenerartorTest {
+
+    private val pdfService = PdfService(testContentPath!!)
+
+    @Throws(IOException::class)
+    private fun getDocument(fixtureName: String): String {
+        return IOUtils.toString(
+            this.javaClass.getResourceAsStream("$PDF_RESOURSE_PATH/$fixtureName"),
+            StandardCharsets.UTF_8
+        )
+    }
+
+    /**
+     * Test if the data in the given byte array represents a PDF file.
+     */
+    fun isPdf(data: ByteArray): Boolean {
+        if (data.size > 4 && data[0] == 0x25.toByte() && // %
+            data[1] == 0x50.toByte() && // P
+            data[2] == 0x44.toByte() && // D
+            data[3] == 0x46.toByte() && // F
+            data[4] == 0x2D.toByte()
+        ) { // -
+
+            // version 1.3 file terminator
+            if (data[5] == 0x31.toByte() && data[6] == 0x2E.toByte() &&
+                data[7] == 0x33.toByte() && data[data.size - 7] == 0x25.toByte() && // %
+                data[data.size - 6] == 0x25.toByte() && // %
+                data[data.size - 5] == 0x45.toByte() && // E
+                data[data.size - 4] == 0x4F.toByte() && // O
+                data[data.size - 3] == 0x46.toByte() && // F
+                data[data.size - 2] == 0x20.toByte() && // SPACE
+                data[data.size - 1] == 0x0A.toByte()
+            ) { // EOL
+                return true
+            }
+
+            // version 1.3 file terminator
+            if (data[5] == 0x31.toByte() && data[6] == 0x2E.toByte() &&
+                data[7] == 0x34.toByte() && data[data.size - 6] == 0x25.toByte() && // %
+                data[data.size - 5] == 0x25.toByte() && // %
+                data[data.size - 4] == 0x45.toByte() && // E
+                data[data.size - 3] == 0x4F.toByte() && // O
+                data[data.size - 2] == 0x46.toByte() && // F
+                data[data.size - 1] == 0x0A.toByte()
+            ) { // EOL
+                return true
+            }
+        }
+        return false
+    }
+
+    @Throws(IOException::class)
+    private fun pdfsAreEqual(resource: String, actualPdfBytes: ByteArray): Boolean {
+        Files.createDirectories(Paths.get(TEST_OUTPUT_PATH))
+
+        // Load expected PDF document from resources, change class below.
+        var expectedPdfBytes: ByteArray
+        PdfGenerartorTest::class.java.getResourceAsStream("$PDF_RESOURSE_PATH/$resource")
+            .use { expectedIs -> expectedPdfBytes = IOUtils.toByteArray(expectedIs) }
```

Comment:
Går å bruke TestUtil her
```suggestion
        var expectedPdfBytes = toByteArray("$PDF_RESOURSE_PATH/$resource")
```

Må slette / fra PDF_RESOURSE_PATH  `private const val PDF_RESOURSE_PATH = "pdf"`også